bolt-modules/boltlib/lib/puppet/functions/write_file.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'tempfile'
+
+# Write contents to a file on the given set of targets.
+#
+# > **Note:** Not available in apply block
+#
+# > **Note:** Files are granted 0600 permissions. If a file contains sensitive content
```

Comment:
What causes this to happen?

Reply:
Removed. I had put that when reading about `Tempfile#create`, but forgot permissions could be different when the file is written to another system.